client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -39,6 +39,17 @@ const DataFilesToolbar = ({ scheme, api }) => {
     )
   );
 
+  const publicData =
+    api !== 'googledrive' && scheme !== 'public' && scheme !== 'community';
```

Comment:
Is `publicData` the best term to use here? Would something like `userData` be more accurate?